.yarnrc.yml

```diff
@@ -5,3 +5,4 @@ plugins:
     spec: "@yarnpkg/plugin-interactive-tools"
 
 yarnPath: .yarn/releases/yarn-3.3.0.cjs
+nodeLinker: node-modules
```

Comment:
Srry another thing I wanted to check is if we can use GitHub actions to add this line to the file, rather than committing it with this line. 

I searched it up and found some possible solutions yesterday, however I haven't found anything too concrete yet. 

Just wanna make sure we don't introduce broken dependencies for pnp.js in the future